common/hogvm/__tests__/__snapshots__/stl.js

```diff
@@ -61,9 +61,10 @@ function toIntervalMonth(val) { return __toHogInterval(val, 'month') }
 function toIntervalDay(val) { return __toHogInterval(val, 'day') }
 function toDateTime (input, zone) { return __toDateTime(input, zone) }
 function toDate (input) { return __toDate(input) }
-function substring(s, start, length = s.length - start + 1) {
+function substring(s, start, optionalLength) {
     if (typeof s !== 'string') return '';
     const startIdx = start - 1;
+    const length = Number.isNaN(optionalLength + 1) ? s.length - startIdx : optionalLength
```

Comment:
Why not just `typeof optionalLength === 'number'` ?

Reply:
That seems like a better idea